tests/tests_pytorch/strategies/test_tpu_spawn.py

```diff
@@ -45,39 +44,10 @@ def predict_dataloader(self):
 _loader_no_len = CustomNotImplementedErrorDataloader(_loader)
 
 
-@pytest.mark.parametrize(
-    "train_dataloaders, val_dataloaders, test_dataloaders, predict_dataloaders",
-    [
-        (_loader_no_len, None, None, None),
-        (None, _loader_no_len, None, None),
-        (None, None, _loader_no_len, None),
-        (None, None, None, _loader_no_len),
-        (None, [_loader, _loader_no_len], None, None),
-    ],
-)
-def test_error_iterable_dataloaders_passed_to_fit(
```

Comment:
This test will be added back in https://github.com/Lightning-AI/lightning/pull/16726/files#diff-6aedcbda38b84cd2299e850355bf10dccc690b4bc71fa8774410269ee56d1bf5L58. Fixing it here would require major changes that will land later.

The test below still tests that the exception is raised.

And the test that will be added asserts that `strategy.process_dataloader` is called as many times as we have dataloaders